README.md

```diff
@@ -7,11 +7,11 @@ A Node.js module for implementing BLE (Bluetooth Low Energy) peripherals.
 
 Need a BLE central module? See [noble](https://github.com/sandeepmistry/noble).
 
-__Note:__ Mac OS X, Linux, and Windows are currently the only supported OSes.
+__Note:__ macOS, Linux, and Windows are currently the only supported OSes.
```

Comment:
Maybe we can have, "Mac OS X/macOS" here?

I might mislead people to think macOS Sierra is needed ... what do you think?